frontend/public/components/secrets/create-secret.tsx

```diff
@@ -877,22 +877,40 @@ const secretFormFactory = (secretType: SecretTypeAbstraction) => {
   }
 };
 
-const SecretLoadingWrapper = props => {
-  if (!props.obj.loaded) {
-    return <LoadingBox />;
-  }
-  const secretTypeAbstraction = toTypeAbstraction(props.obj.data);
-  const SecretFormComponent = secretFormFactory(secretTypeAbstraction);
-  const fixed = _.reduce(props.fixedKeys, (acc, k) => ({...acc, k: _.get(props.obj.data, k)}), {});
-  return <StatusBox {...props.obj}>
```

Comment:
I think this will break error handling if the secret doesn't exist.

Reply:
not really sure how, since the StatusBox component is mainly operating with `loaded` and `loadError` properties, that are supplied by Firehose